actix-session/src/session.rs

```diff
@@ -77,6 +79,7 @@ impl Default for SessionStatus {
 struct SessionInner {
     state: HashMap<String, String>,
     status: SessionStatus,
+    session_key: SessionKey,
```

Comment:
This must be an `Option`, since the key will only exist if the user has an active session.

Reply:
It should be populated when https://github.com/actix/actix-extras/blob/1774b8a36ef145bb85e2c0db50c9ff78f844804e/actix-session/src/middleware.rs#L218 is called.

Reply:
and this logic needs to be replicated in session.rs? This is tricky for me to understand. This is populated within `get_session_key` fn I'm proposing?